libraries/common/effekt.effekt

```diff
@@ -407,6 +407,19 @@ extern pure def exp(x: Double): Double =
   chez "(exp ${x})"
   ml "Math.exp ${x}"
 
+def pow(base: Double, exponent: Int): Double = {
+  def loop(acc: Double, exponent: Int): Double = {
+    if (exponent == 0) acc
+    else loop(acc * base, exponent - 1)
+  }
+  loop(1.0, exponent)
+}
```

Comment:
We can also make this slightly faster (logarithmic instead of linear) by using a classic algorithm:
```suggestion
def pow(base: Double, exponent: Int): Double = {
  def loop(base: Double, exponent: Int, acc: Double): Double = {
    if (exponent == 0) acc
    else if (exponent.mod(2) == 0) loop(base * base, exponent / 2, acc)  // even exponent
    else loop(base, exponent - 1, acc * base)  // odd exponent
  }
  loop(base, exponent, 1.0)
}
```